voidcrew/modules/cloning/code/cloning_circuitboards.dm

```diff
@@ -0,0 +1,46 @@
+//Computer
+
+/obj/item/circuitboard/computer/prototype_cloning
+	name = "Prototype Cloning (Computer Board)"
+	icon_state = "medical"
+	build_path = /obj/machinery/computer/prototype_cloning
+
+/obj/item/circuitboard/computer/cloning
+	name = "Cloning (Computer Board)"
+	icon_state = "medical"
+	build_path = /obj/machinery/computer/cloning
+
+//Machinery
+
+/obj/item/circuitboard/machine/clonepod
+	name = "Clone Pod (Machine Board)"
```

Comment:
I think 'Machine Board' is handled by parent, we might not need this? Should be tested or looked into because I might be wrong.